pkg/operation/botanist/component/resourcemanager/resource_manager_test.go

```diff
@@ -399,7 +399,7 @@ var _ = Describe("ResourceManager", func() {
 						SyncPeriod:      &healthSyncPeriod,
 					},
 					KubeletCSRApprover: resourcemanagerv1alpha1.KubeletCSRApproverControllerConfig{
-						Enabled:         true,
+						Enabled:         true && !isWorkerless,
```

Comment:
```suggestion
						Enabled:         !isWorkerless,
```

same for other ocurrences